src/containers/LanguageToggleContainer.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { useTranslation, useI18next } from 'gatsby-plugin-react-i18next';
+import Switch from '../components/site/switch';
+
+const LanguageToggleContainer = () => {
+  const { t } = useTranslation('interface');
+  const { useState, useEffect, useRef } = React;
+  const {
+    language: currentLanguage,
+    defaultLanguage,
+    changeLanguage,
+  } = useI18next();
+  const [toEs, setToEs] = useState(currentLanguage !== defaultLanguage);
+  const switchCount = useRef(0);
+  const id = 'langSelect';
+
+  useEffect(() => {
+    // don't run effect for change to `toEs` on mount
+    if (switchCount.current > 0) {
+      const newLang = toEs ? 'es' : 'en';
+      changeLanguage(newLang);
+    }
+
+    switchCount.current += 1;
+  }, [toEs]);
+
+  const handleClick = () => {
+    setToEs(prevValue => !prevValue);
+  };
+
+  return (
+    <fieldset className="language-toggle-container">
```

Comment:
based all of this on the rubin-obs language toggle with the exception that I used the built-in `changeLanguage` method in i18next and instead of using styled components built up HTML elements and a new React component